Light.json

```diff
@@ -3176,6 +3176,7 @@
         "type": "Character",
         "subType": "Droid",
         "uniqueness": "*",
+        "destiny": "1 or 6",
```

Comment:
Please use the high number of `6` as the destiny.
For the low number of `1`, please place in a field of `uniqueDestiny: "1"`